GVRf/Framework/src/org/gearvrf/GVRRenderTexture.java

```diff
@@ -52,6 +52,31 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height,
         super(gvrContext, NativeRenderTexture.ctorMSAA(width, height,
                 sampleCount));
 
+    /**
```

Comment:
@danke-sra The Javadoc is *inside* the constructor. Something happened with the merging. This is how it looks:

```
 37     /**
 38      * Constructs a GVRRenderTexture for a frame buffer of the specified size,
 39      * with MSAA enabled at the specified sample count.
 40      *
 41      * @param gvrContext
 42      *            Current gvrContext
 43      * @param width
 44      *            Width of the frame buffer.
 45      * @param height
 46      *            Height of the frame buffer.
 47      * @param sampleCount
 48      *            MSAA sample count.
 49      */
 50     public GVRRenderTexture(GVRContext gvrContext, int width, int height,
 51             int sampleCount) {
 52         super(gvrContext, NativeRenderTexture.ctorMSAA(width, height,
 53                 sampleCount));
 54 
 55     /**
 56      * Constructs a GVRRenderTexture for a frame buffer of the specified size,
 57      * with MSAA enabled at the specified sample count, and with specified color
 58      * format, depth format, resolution depth and texture parameters.
 59      *
 60      * @param gvrContext
 61      *            Current gvrContext
 62      * @param width
 63      *            Width of the frame buffer.
 64      * @param height
 65      *            Height of the frame buffer.
 66      * @param sampleCount
 67      *            MSAA sample count.
 68      * @param colorFormat
 69      *            GVR color format. See {@linkplain org.gearvrf.utility.
 70      *            VrAppSettings.EyeBufferParams.ColorFormat ColorFormat}.
 71      * @param depthFormat
 72      *            Depth format. See {@linkplain org.gearvrf.utility.
 73      *            VrAppSettings.EyeBufferParams.DepthFormat DepthFormat}.
 74      * @param resolveDepth
 75      *            If true, resolves the depth buffer into a texture.
 76      * @param parameters
 77      *            Texture parameters. See {@link GVRTextureParameters}.
 78      *
 79      */
 80         mWidth = width;
 81         mHeight = height;
 82     }
```